src/views/TabView/TabBarBottom.js

```diff
@@ -67,6 +73,7 @@ class TabBarBottom extends React.PureComponent<Props> {
     showLabel: true,
     showIcon: true,
     allowFontScaling: true,
+    adaptiveTabs: canUseHorizontalTabs,
```

Comment:
Change the default value to `isIOS11 && isIPad`

Reply:
Apple uses horizontal tabs on iPhones as well by default, so I'm just going to put the default as isIOS11